frontend/src/components/kl-delete-all.js

```diff
@@ -1,8 +1,8 @@
-import api from '../../../config/api.json';
+import API from 'api.json';
```

Comment:
hmm.. alias for the `api.json` already exists, or do you mean path `config/api.json`?